python/tests/test_shared_pipeline_stress.py

```diff
@@ -0,0 +1,29 @@
+import unittest
+
+from tests.shared_test_pipeline import SharedTestPipeline
+from tests import TEST_CLIENT
+
+
+class TestPipeline(SharedTestPipeline):
+    result = None
+
+    def test_create_pipeline(self):
+        """
+        CREATE TABLE tbl(id INT) WITH ('materialized' = 'true');
+        CREATE MATERIALIZED VIEW v0 AS SELECT * FROM tbl;
+        """
+        pass
+
+    def test_get_pipeline_logs(self):
+        self.pipeline.start()
+        # regression test for https://github.com/feldera/feldera/issues/4394
+        for _ in range(200):
+            logs = TEST_CLIENT.get_pipeline_logs(self.pipeline.name)
+            start = next(logs)
+            assert "Fresh start of pipeline log" in start
+        TEST_CLIENT.pause_pipeline(self.pipeline.name)
```

Comment:
You should also be able to use `self.pipeline` to use the higher level API instead of using the `TEST_CLIENT` directly.